DESCRIPTION

```diff
@@ -23,6 +23,9 @@ Imports:
     tidyr
 Suggests:
     cowplot,
+    visR,
+    envsetup,
+    config
```

Comment:
There's a `,` missing here leading the `R CMD check` workflow to fail.

Reply:
resolved